13/umbraco-cms/reference/content-delivery-api/property-expansion-and-limiting.md

```diff
@@ -0,0 +1,1376 @@
+---
+description: Using property expansion and limiting to shape the Delivery API output
+---
+
+# Property expansion and limiting
+
+{% hint style="info" %}
+This article explains the mechanics of property expansion and limiting in depth. If you haven't already, please read the [Getting started](https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api) article first - in particular the "Concepts" section.
+{% endhint %}
+
+Property expansion and limiting applies only to select property editors. The following built-in property editors in Umbraco support expansion and limiting:
+
+- Picker editors
+  - Content Picker
+  - Media Picker
+  - Media Picker (legacy)
+  - Multinode Treepicker
+- Block-based editors
+  - Block List
+  - Block Grid
+  - Rich Text Editor (with blocks)
+
+{% hint style="info" %}
+When working with property expansion and limiting in API queries, there are two rules of thumb to keep in mind:
+
+1. Expandable properties are _not_ expanded by default. They must be expanded explicitly.
+2. All properties are included in the API output by default. We can apply limiting to limit the included properties.
+{% endhint %}
+
+## Working with picker editors
+
+In the following, we will be querying a content tree with blog posts and blog authors:
```

Comment:
```suggestion
In the following example, we will be querying a content tree with blog posts and blog authors:
```

Reply:
@kjac Seemed like something was missing here, so i added this suggestion :-)

Reply:
We should do plural then... the querying goes on for several examples. I'll amend 😄